imports/plugins/core/checkout/client/components/addEmail.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Meteor } from "meteor/meteor";
+import { Orders } from "/lib/collections";
+import { Reaction, i18next } from "/client/api";
+import { Components } from "@reactioncommerce/reaction-components";
+
+
+class AddEmail extends React.Component {
```

Comment:
So we have both `Component` and `Components` in the namespace. That seems unfortunate?